Assets/Input_Test/Scripts/02_Input/Gamepad4InputManager.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Gamepad4InputManager : MonoBehaviour
```

Comment:
Naming seems somewhat confusing as there's also an InputManager class in the scripts. `GamepadForOldInputSystem` and `GamepadForNewInputSystem` seem more obvious.